opentech/apply/funds/urls.py

```diff
@@ -35,6 +36,7 @@
 submission_urls = ([
     path('', SubmissionOverviewView.as_view(), name="overview"),
     path('all/', SubmissionListView.as_view(), name="list"),
+    re_path('documents/(.*)$', SubmissionPrivateMediaRedirectView.as_view(), name='private_media_redirect'),
```

Comment:
Can you not have a better path matching here rather than having to handling the 404 etc in the view itself?